### DISCUSSION TITLE
lost files while they were pushed on github private repository

### DISCUSSION THREAD

**Discussion by Kagame11** [community/community#88670] [Repositories]
### Select Topic Area

Show & Tell

### Body

I have pushed my project on GitHub as a private repository before selling my macbook after selling my mac I baught a new laptop with window OS while i was cloning my repository on GitHub to use it again on my new PC  I saw only two files i.e gitattributes and README files only what would i do to gain back my whole project files

**Comment by ghost** [comment]
check your commit history and GitHub repository to ensure your project files are indeed present

**Reply by ghost** [reply]
Via this command

```shell
git log
```

**Reply by ghost** [reply]
Make sure you are cloning the correct repository and branch:
```shell
git clone <repository_url>
cd <repository_directory>
```

**Reply by ghost** [reply]
If the project files are still not visible, you might want to check the contents of the repository on the GitHub website to see if the files are present there.

**Reply by ghost** [reply]
If you really need it in you local machine but you can see it in website and not while cloning then try it to download it has .zip file
